contents/docs/product-analytics/identify.mdx

```diff
@@ -81,133 +81,106 @@ The ID returned is either the ID automatically generated by PostHog or the ID th
 
 ## Alias: Assigning multiple distinct IDs to the same user 
 
-Sometimes, you want to assign multiple distinct IDs to a single user. For example, if a distinct ID which is typically used on the frontend is not available in certain parts of your backend code, you can use `alias` to connect the frontend distinct ID to one accessible on the backend. This will merge all past and future events into the same user.
+Sometimes, you want to assign multiple distinct IDs to a single user.
+- Your user may have a different distinct ID known on the frontend and the backend. 
+- Your user maybe unauthenticated on the browser, but authenticated in your app, and you want to associate the two sessions.
+- Your user adds items to their cart before logging in with an anonymous distinct ID, logs in at checkout, and you want to associate the two sessions.
 
-In the below example, we assign the user with `frontend_id` another ID: `backend_id`. This means that any events submitted using either `frontend_id` or `backend_id` will be associated with the same user.
```

Comment:
I think we need to keep something like this, what is `alias_id` and what is `distinct_id`. This is really confusing without an explanation.

Reply:
Most people don't know what the word alias means.

Reply:
- Which one should people use going forward?
- What happens when you capture an event with either of the IDs?

Reply:
The problem is. They're not different in code. They're both stored as distinct IDs that point to the same person. They're practically identical, and which is kept as the distinct ID afaik isn't super clear (it depends with heavy ***)

I'll see if there's another example we can keep in, but the original example isn't matching behavior.

Reply:
If you wanted to alias more IDs to one of the IDs, which would you use? Is there any limitations?